### PR TITLE
Added DDEX to DApps

### PR DESCRIPTION
**Pull request by andytudhope**
### Summary:

Added DDEX to the DApps list as I could not reproduce UX issues entirely and their team has committed to ensuring that it works fully in Status.

### Review notes:
Please make sure to check that you can specify the limits and amounts correctly on trades, as Rachel was having issues with this.

### Steps to test:
- Open Status
- navigate to Open a DApp.
- Test trading on DDEX.

status: ready

### CODE REVIEW

File: resources/default_contacts.json [hunk 1]
```diff
@@ -108,6 +108,13 @@
         "dapp?": true,
         "photo-path": "contacts://name-bazaar"
     },
+    "ddex": {
+        "name" : {"en" : "DDEX "},
+        "description": "The most user-friendly decentralized exchange for Ethereum and ERC-20 tokens. ",
```
Comment (dmitryn) [inline resources/default_contacts.json]: Extra space at the end

**Review by dmitryn — APPROVED** [review]

**Review by jeluard — APPROVED** [review]

**Comment by lukaszfryc** [comment]
Here are some issues I've found:

1. It does not work on iPhone 8 (iOS 11.2.6). Nothing happens when I click "Start Trading"

2. Part of my address is cut


3. When I tested it on Android on Ropsten, I could not finalize my transaction. After clicking "Buy", "loading accounts balances" was shown forever.

 
@andytudhope how can I contact DDEX to ask about these issues? Can you refer them in this PR?

**Comment by rachelhamlin** [comment]
On iPhone 6s Plus, my address is also cut.

Additionally, this is my view when entering a limit price or amount:

image uploaded from ios

**Comment by lukaszfryc** [comment]
@andytudhope any news from Bowen?

**Comment by andytudhope** [comment]
No response yet. Let's give it 1 or 2 more days and then close here if he doesn't get back to us.

**Comment by lukaszfryc** [comment]
@andytudhope ping

**Comment by andytudhope** [comment]
No update after 2 emails to Bowen. Closing.

**Pull request closed without merging**